torchci/rockset/metrics/__sql/workflow_load.sql

```diff
@@ -13,6 +13,7 @@ WHERE
     PARSE_TIMESTAMP_ISO8601(workflow.created_at) >= PARSE_DATETIME_ISO8601(:startTime)
     AND PARSE_TIMESTAMP_ISO8601(workflow.created_at) < PARSE_DATETIME_ISO8601(:stopTime)
     AND workflow.name IN ('pull', 'trunk', 'nightly', 'periodic')
+    AND workflow.repository.full_name = 'pytorch/pytorch'
```

Comment:
This doesn't include stuff from pytorch/pytorch-canary, which still uses our infra, so I'm not sure if this is right.